docs/en/test_cards/mc_07_optical_flow_failure.md

```diff
@@ -0,0 +1,44 @@
+# Test MC_07 - Optical Flow Failure
+
+## Objective
+
+To test that optical flow works as expected with a low mounted optical flow sensor
+
+## Preflight
+
+Ensure that the drone's optical flow sensor is mounted less than an inch off of the ground.
+
+Disconnect all GPS / compasses and ensure vehicle is using optical flow for navigation
+([Setup Information here](../sensor/optical_flow.md))
+
+Ensure there are no other sources of positioning besides optical flow
+
+- [EKF2_OF_CTRL](../advanced_config/parameter_reference.md#EKF2_OF_CTRL): `1`
+- [EKF2_GPS_CTRL](../advanced_config/parameter_reference.md#EKF2_GPS_CTRL): `0`
+- [EKF2_EV_CTRL](../advanced_config/parameter_reference.md#EKF2_EV_CTRL): `0`
+- [SYS_HAS_MAG](../advanced_config/parameter_reference.md#SYS_HAS_MAG): `0`
+
+Ensure that the drone can go into Position flight mode while still on the ground
+
+## Flight Tests
+
+❏ Position flight mode
+
+&nbsp;&nbsp;&nbsp;&nbsp;❏ Horizontal position should hold current value with stick centered
+
+&nbsp;&nbsp;&nbsp;&nbsp;❏ Vertical position should hold current value with stick centered
+
+&nbsp;&nbsp;&nbsp;&nbsp;❏ Throttle response set to climb/descent rate
+
+&nbsp;&nbsp;&nbsp;&nbsp;❏ Pitch/Roll/Yaw response set to pitch/roll/yaw rates
+
+## Landing
+
+❏ Land in either Position mode with the throttle below 40%
+
+❏ Upon touching ground, copter should disarm automatically within 2 seconds (default: see [COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND))
```

Comment:
make sure MPC_THR_MIN is set correctly. If too low land_detector will never detect landing

Reply:
added to the top